Index: dom/src/base/nsDOMClassInfo.cpp
===================================================================
RCS file: /cvsroot/mozilla/dom/src/base/nsDOMClassInfo.cpp,v
retrieving revision 1.291
diff -u -9 -p -r1.291 nsDOMClassInfo.cpp
--- dom/src/base/nsDOMClassInfo.cpp	30 Jul 2005 20:57:05 -0000	1.291
+++ dom/src/base/nsDOMClassInfo.cpp	2 Aug 2005 01:31:30 -0000
@@ -8126,19 +8126,19 @@ nsHTMLExternalObjSH::GetProperty(nsIXPCo
     if (JSVAL_IS_STRING(id)) {
       *_retval = ::JS_GetUCProperty(cx, pi_obj, id_chars, id_length, vp);
     } else {
       *_retval = ::JS_GetElement(cx, pi_obj, JSVAL_TO_INT(id), vp);
     }
 
     return *_retval ? NS_SUCCESS_I_DID_SOMETHING : NS_ERROR_FAILURE;
   }
 
-  return nsElementSH::GetProperty(wrapper, cx, obj, id, vp, _retval);
+  return NS_OK;
 }
 
 NS_IMETHODIMP
 nsHTMLExternalObjSH::SetProperty(nsIXPConnectWrappedNative *wrapper,
                                  JSContext *cx, JSObject *obj, jsval id,
                                  jsval *vp, PRBool *_retval)
 {
   JSObject *pi_obj = ::JS_GetPrototype(cx, obj);
 

