diff -du ./config.h /home/harald/troll/qt-3.3/src/3rdparty/sqlite/config.h
--- ./config.h	2004-03-30 19:13:22.000000000 +0200
+++ /home/harald/troll/qt-3.3/src/3rdparty/sqlite/config.h	2004-01-08 14:29:23.000000000 +0100
@@ -1 +1,23 @@
-#define SQLITE_PTR_SZ 4
+#include <qglobal.h>
+#include <qconfig.h>
+
+#ifndef QT_POINTER_SIZE
+# ifdef Q_OS_WIN32
+#  define QT_POINTER_SIZE 4
+# elif Q_OS_WIN64
+#  define QT_POINTER_SIZE 8
+# else
+#  error This platform is unsupported
+# endif
+#endif /* QT_POINTER_SIZE */
+
+#define SQLITE_PTR_SZ QT_POINTER_SIZE
+
+#ifdef UNICODE
+# undef UNICODE
+#endif
+
+#ifdef Q_CC_MSVC
+# pragma warning(disable: 4018)
+# pragma warning(disable: 4761)
+#endif
diff -du ./os.h /home/harald/troll/qt-3.3/src/3rdparty/sqlite/os.h
--- ./os.h	2004-03-30 19:13:19.000000000 +0200
+++ /home/harald/troll/qt-3.3/src/3rdparty/sqlite/os.h	2004-01-08 14:29:23.000000000 +0100
@@ -17,6 +17,7 @@
 #ifndef _SQLITE_OS_H_
 #define _SQLITE_OS_H_
 
+#include "config.h"
 /*
 ** Helpful hint:  To get this to compile on HP/UX, add -D_INCLUDE_POSIX_SOURCE
 ** to the compiler command line.
